pkgs/applications/misc/pop-launcher/default.nix

```
@@ -0,0 +1,41 @@
+{ rustPlatform
+, fetchFromGitHub
+, lib
+, just
+, runtimeShell
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pop-launcher";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "launcher";
+    rev = "${version}";
+    sha256 = "sha256-BQAO9IodZxGgV8iBmUaOF0yDbAMVDFslKCqlh3pBnb0=";
+  };
+
+  postPatch = ''
+    substituteInPlace justfile \
+        --replace '#!/usr/bin/env sh' '#!${runtimeShell}'
+    substituteInPlace src/lib.rs \
+        --replace '/usr/lib/pop-launcher' "$out/share/pop-launcher"
+    substituteInPlace plugins/src/scripts/mod.rs \
+        --replace '/usr/lib/pop-launcher' "$out/share/pop-launcher"
+  '';
+
+  cargoSha256 = "sha256-cTvrq0fH057UIx/O9u8zHMsg+psMGg1q9klV5OMxtok=";
+
+  nativeBuildInputs = [ just ];
+
+  buildPhase = "just";
+  installPhase = "just base_dir=$out/ install";
```

```suggestion
  buildPhase = ''
    just
  '';

  installPhase = ''
    just base_dir=$out/ install
  '';
```
Do we have thread support like makes -j?

---

> Do we have thread support like makes -j?

As far as I can tell, that isn't supported yet (See: https://github.com/casey/just/issues/626)